src/hooks/useVotesInformation.ts

```diff
@@ -0,0 +1,30 @@
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+
+import { Governance } from '../api/Governance'
+import { Snapshot } from '../api/Snapshot'
+import { SNAPSHOT_SPACE } from '../entities/Snapshot/constants'
+import { MatchResult, calculateMatch } from '../entities/Snapshot/utils'
+
+
+export default function useVotesInformation(userAccount: string | null, otherAccount: string | null) {
```

Comment:
Maybe `useVotesComparison`? Or something that indicates this is comparing between two addresses or accounts.